Makefile

```diff
@@ -41,11 +41,13 @@ sass:
 install-test:
 	pip install -q -r requirements/test.txt
 
-install-sys-requirements: install-system install-node
+install-sys-requirements: install-system install-node install-sass
```

Comment:
Not strictly related to this PR, but @dianakhuang since you're rebuilding your dev environment, I had to do this to get the installation steps on https://openedx.atlassian.net/wiki/display/TNL/ORA+FAQ to succeed.

I suspect it's related to the ruby removal that happened last week, but idk for sure.